src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -12,19 +1,61 @@
+import React from 'react';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  status: string;
+  query: string;
+  onStatusChange: (value: string) => void;
+  onQuerryChange: (value: string) => void;
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+export const TodoFilter: React.FC<Props> = ({
+  status,
+  query,
+  onStatusChange,
+  onQuerryChange,
```

Comment:
The prop `onQuerryChange` is misspelled. It should be `onQueryChange` to match the corrected prop name in the `Props` type definition.